android-whitelist.json

```
@@ -1376,6 +1376,11 @@
       "group": "com\\.mercadolibre\\.android\\.mldashboard",
       "name": "mldashboard",
       "version": "1\\.\\+"
+    },
+    {
+      "group": "com\\.smartlook\\.recording", 
+      "name": "smartlook",
+      "version": "1\\.5\\.2-native"
```

```suggestion
      "version": "1\\.5\\.2\\-native"
```
**Creo** que CI está fallando porque aquí debería estar escapeado.
¿Podemos probar eso?